modules/nlu/src/backend/pipelines/intents/svm_classifier.ts

```diff
@@ -213,14 +227,24 @@ export default class SVMClassifier {
     }
 
     const svm = new this.toolkit.SVM.Trainer({ kernel: 'LINEAR', classifier: 'C_SVC' })
+    const oosSvm = new this.toolkit.SVM.Trainer({ kernel: 'RBF', classifier: 'C_SVC' })
```

Comment:
Now that we have OOS rbf model + L0 being linear. I think it could make sens to use a RBF classifier to scope L1 but that would require even more trials